handsontable/src/helpers/dom/element.js

```diff
@@ -1148,3 +1173,45 @@ export function runWithSelectedContendEditableElement(element, callback, invisib
 
   removeContentEditableFromElementAndDeselect(element, invisibleSelection);
 }
+
+/**
+ * Creates a div element and appends it to the parent element with the provided class name(s) and attributes.
+ *
+ * @private
+ * @param {HTMLElement | null} parentElement The parent element.
+ * @param {string} elementType The type of the new element.
+ * @param {Array[] | undefined} attributes An array containing the attributes to be added. Each element of the array
+ * should be an array in a form of `[attributeName, attributeValue]`.
+ * @param {string|Array} className Class name as string or array of strings.
+ * @returns {HTMLElement} The created div element.
+ */
+export function appendElement(parentElement, elementType, attributes = [], className = []) {
```

Comment:
Can I ask to add a test for that helper?

I'm just thinking out loud and I wonder if it would be more future-proof to use a configuration object instead of arguments. WDYT?

Something like
```js
appendElement(parentElement, { tag, classes, attrs }) {}
// or 
appendElement(parentElement, { tagName, className, attributes }) {}
```
Example usage with code snippet used in this PR
```js
appendElement(TH, {
  tag: 'div',
  attrs: attributesToAdd,
  classes: AFTER_INDICATOR_CLASSNAME,
});
```